site/docs/reference/Connectors/materialization-connectors/Firebolt.md

```diff
@@ -0,0 +1,83 @@
+# Firebolt
+
+This Flow connector materializes [delta updates](../../../concepts/materialization.md#delta-updates) of Flow collections into Firebolt `FACT` or `DIMENSION` tables.
+
+To interface between Flow and Firebolt, the connector creates an _external table_:
+a virtual table in an S3 bucket used as a staging area.
+
+[`ghcr.io/estuary/materialize-firebolt:dev`](https://ghcr.io/estuary/materialize-firebolt:dev) provides the latest connector image. You can also follow the link in your browser to see past image versions.
+
+## Prerequisites
+
+To use this connector, you'll need:
+
+* A Firebolt database with at least one [engine](https://docs.firebolt.io/working-with-engines/working-with-engines.html)
+* An S3 bucket in which to stage the external table
+* At least one Flow [collection](../../../concepts/collections.md)
+
+:::tip
+ If you haven't yet captured your data from its external source,
+ start at the beginning of the [guide to create a dataflow](../../../guides/create-dataflow.md).
+ You'll be referred back to this connector-specific documentation at the appropriate steps.
+:::
+
+## Configuration
+
+To use this connector, begin with data in one or more Flow collections.
+Use the below properties to configure a Firebolt materialization, which will direct Flow data to your desired Firebolt tables via an external table.
+
+### Properties
+
+#### Endpoint
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| `/aws_key_id` | AWS Key ID | AWS Key ID for accessing the S3 bucket. | string |  |
+| `/aws_region` | AWS Region | AWS Region the bucket is in. | string |  |
+| `/aws_secret_key` | AWS Secret Key | AWS Secret Key for accessing the S3 bucket. | string |  |
+| **`/database`** | Database | Name of the Firebolt database. | string | Required |
+| **`/engine_url`** | Engine URL | Engine URL of the Firebolt database, in the format: `<engine-name>.<organization>.<region>.app.firebolt.io`. | string | Required |
```

Comment:
Did the title and description end up being auto-generated for Firebolt or were they still missing?

Reply:
They were auto-generated! Just tweaked a couple small things for formatting